packages/oui-stepper/README.md

```diff
@@ -0,0 +1,251 @@
+    </oui-step-form>
+
+    <oui-step-form header="Step 2" skippable
+        description="This is a description">
+        <oui-field label="Firstname" help-text="At least 3 chars" size="xl">
+            <input class="oui-input" type="text" name="firstname"
+                ng-model="$ctrl.user.firstname" maxlength="32">
+        </oui-field>
+        <oui-field label="Lastname" help-text="At least 3 chars" size="xl">
+            <input class="oui-input" type="text" name="lastname"
+                ng-model="$ctrl.user.lastname" maxlength="32">
+        </oui-field>
+        <oui-field label="Description" size="xl">
+            <oui-textarea model="$ctrl.user.description" name="description" placeholder="Please insert your text..." maxlength="10"></oui-textarea>
+        </oui-field>
+    </oui-step-form>
+
+    <oui-step-form header="Step 3"
+        description="This is a description">
+        <oui-field label="OS" size="m">
+            <label class="oui-select">
+                <select name="os" ng-model="os" class="oui-select__input" required>
+                    <option ng-value="undefined">Select the OS</option>
+                    <option value="freebsd">FreeBSD</option>
+                    <option value="linux">Linux</option>
+                    <option value="osx">OSX</option>
+                    <option value="windows">Windows</option>
+                </select>
+                <i class="oui-icon oui-icon-chevron-down" aria-hidden="true"></i>
+            </label>
+        </oui-field>
+        <oui-field label="Server options">
+            <oui-checkbox name="ssl" text="SSL" model="$ctrl.ssl" required></oui-checkbox>
+            <oui-checkbox name="hsts" text="HSTS" model="$ctrl.hsts"></oui-checkbox>
+        </oui-field>
+    </oui-step-form>
+</oui-stepper>
+```
+
+## API
+
+### oui-stepper
+
+| Attribute       | Type            | Binding | One-time binding | Values                 | Default             | Description                                   |
+| ----            | ----            | ----    | ----             | ----                   | ----                | ----                                          |
+| `name`          | string          | @?      | yes              |                        |                     | stepper name used to identify step            |
+| `id`            | string          | @?      | yes              |                        |                     | stepper id used to identify step              |
+| `linear`        | boolean         | <?      | yes              |                        | false               | enable stepper to handle steps                |
+| `on-init`       | function        | &       |                  |                        |                     | initialization function                       |
+| `on-finish`     | function        | &       |                  |                        |                     | submit all steps function                     |
+
```

Comment:
A configuration section for provider could be interesting.
Like field doc http://build_oui_stepper.ui-kit.ovh/#!/oui-angular/field

Reply:
Nice catch ! I forgot this part...

Reply:
Done